src/lang/bg.json

```diff
@@ -2,5 +2,22 @@
   "welcome": "Добре дошли.",
   "projects": {
     "title": "Проекти"
-  }
-}
+  },
+  "errors": {
+    "email": {
+      "required": "Полето за имейл e задължително",
+      "invalid": "Невалиден имейл адрес"
+    },
+    "password": {
+      "required": "Полето за парола e задължително",
+      "short": "Паролата трябва да e поне 4 символа",
+      "long": "Паролата трябва до 10 символа",
+      "digit": "Паролата трябва да съдържа поне една цифра",
+      "specialCharacter": "Паролата трябва да съдържа поне един специален знак"
+    }
+  },
+  "description": "Вътрешна система за менажиране на доставки",
```

Comment:
put this inside `login` so we know they are for `login`.
In short do this:

```
"login": {
  "description": "Вътрешна система за менажиране на доставки",
  "title": "Вход",
}
```

Translations that are for a specific page should be into its own object so we know what this title is for and etc. 

And actually lets also do this for the input labels so they are again into their own block sort to say and its easier to know what this is for.

```
"label": {
  "email": "Имейл",
  "password": "Парола"
}
```

Reply:
Fixed